Makefile

```diff
@@ -35,10 +35,35 @@ check-clean-repo:
 update-prow:
 	(cd prow; sh update.sh)
 
-lint:
+lint: shellcheck checkconfig check-resource-limits
+
+shellcheck:
 	find . -name '*.sh' -print0 | xargs -0 -r shellcheck
+
+checkconfig:
 	checkconfig -strict -config-path prow/config.gen.yaml -plugin-config prow/plugins.yaml
 
+check-resource-limits:
+	@fail=0;\
```

Comment:
put this in a place like `scripts/check-resource-limits.sh` and just invoke it here.

Reply:
+1 this started out quite simple but quickly grew out of proportion